lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
 
+    //Const for main stylesheet.
     const stylesheet = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
-    stylesheet.rel = 'stylesheet';
     stylesheet.href = 'chrome://universalsearch-skin/content/style.css';
+    stylesheet.rel = 'stylesheet';
     stylesheet.type = 'text/css';
     stylesheet.id = 'universal-search-stylesheet';
     stylesheet.style.display = 'none';
 
+    //Const for platform-specific stylesheet.
+    const stylesheetPlatform = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
+    stylesheetPlatform.rel = 'stylesheet';
+    stylesheetPlatform.type = 'text/css';
+    stylesheetPlatform.id = 'universal-search-stylesheetPlatform';
+    stylesheetPlatform.style.display = 'none';
+
+    // If OS is Mac, load style-osx.css.
+    if (osString == 'Darwin') {
```

Comment:
To match the rest of the codebase, let's use the strict triple-equals for these comparisons, `===`, rather than `==`.